base_restrict_user_fields/security/restrict_user_fields.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="group_restrict_fields" model="res.groups">
+            <field name="name">Not restrict sensitive fields</field>
```

Comment:
Better this naming (take in account this group will appear in a general and heterogeneous list):
```suggestion
            <field name="name">User management: don't restrict sensitive fields</field>
```